WeakAurasOptions/TriggerOptions.lua

```diff
@@ -115,6 +117,34 @@ local function AddOptions(allOptions, data)
     end
   end
 
+  triggerOptions["addTriggerOption"] = {
+    __title = L["Add Trigger"],
+    __order = 5000,
+    __withoutheader = true,
+    __topLine = true,
+    __collapsed = false,
+    addTrigger = {
+      type = "execute",
+      width = WeakAuras.normalWidth,
+      name = L["Add Trigger"],
```

Comment:
At first look i though there was an issue with the add button missing, I think this button should include the green "+" texture Interface\\AddOns\\WeakAuras\\Media\\Textures\\add

Reply:
All the other Add buttons don't have that. I think that's just somethign to get used to.